uniffi_bindgen/src/bindings/python/templates/ExternalTemplate.py

```diff
@@ -0,0 +1,13 @@
+{%- let name = self.name() %}
+{%- let crate_name = self.crate_name() %}
+
+class FfiConverter{{ name }}(FfiConverterUsingByteBuffer):
+    @staticmethod
+    def _write(value, buf):
+        from {{ crate_name|fn_name_py }} import {{ name }};
+        {{ name }}._write(value, buf)
```

Comment:
I initially had some trouble getting the tests to work, but is is what I ended up with.
But you are right that this will break down if the underlying types are anything different.

Let's explore that in followups when we expand on external types.